code/game/gamemodes/cult/cult.dm

```diff
@@ -75,18 +99,40 @@
 	return (cult.len>=required_enemies)
 
 
+/datum/game_mode/cult/proc/memorize_cult_objectives(datum/mind/cult_mind)
+	for(var/obj_count = 1,obj_count <= cult_objectives.len,obj_count++)
+		var/explanation
+		switch(cult_objectives[obj_count])
+			if("survive")
+				explanation = "Our knowledge must live on. Make sure at least [acolytes_needed] acolytes escape on the shuttle to spread their work on an another station."
+			if("sacrifice")
+				if(sacrifice_target)
+					explanation = "Sacrifice [sacrifice_target.name], the [sacrifice_target.assigned_role]. You will need the Sacrifice rune and three acolytes to do so."
+				else
+					explanation = "Free objective."
+			if("eldergod")
+				explanation = "Summon Nar-Sie via the rune 'Call Forth The Geometer'. It will only work if nine acolytes stand on and around it."
+		cult_mind.current << "<B>Objective #[obj_count]</B>: [explanation]"
+		cult_mind.memory += "<B>Objective #[obj_count]</B>: [explanation]<BR>"
+
 /datum/game_mode/cult/post_setup()
 	modePlayer += cult
-	for(var/datum/mind/cult_mind in cult)
```

Comment:
Why did you remove this code block? This is what gave the original cultists their hud, objectives, gear, etc.
Move the objective generation code above this.